docs/source/teams/installation.rst

```diff
@@ -269,15 +281,24 @@ you simply need to provide
 `service account credentials <https://cloud.google.com/iam/docs/service-accounts>`_
 to your Teams client with read access to the relevant objects and buckets.
 
-You can register GCP credentials on a particular machine by adding the
-following key to your :ref:`media cache config <teams-media-cache-config>`:
+You can do this in any of the following ways:
+
+1. Permanently register GCS credentials on a particular machine by adding the
+following keys to your :ref:`media cache config <teams-media-cache-config>`:
 
 .. code-block:: json
 
     {
         "google_application_credentials": "/path/to/gcp-service-account.json"
     }
 
+2. Provide GCS credentials on a per-session basis by setting the following
+environment variables to point to your GCS credentials on disk:
+
+    .. code-block:: shell
+
+        export GOOGLE_APPLICATION_CREDENTIALS="/path/to/gcp-service-account.json"
```

Comment:
Not sure if this is the only way to provide credentials to [gcs.Client()](https://github.com/voxel51/eta/blob/eed37df10661d504485fd06ff68274ad203f682a/eta/core/storage.py#L1893). Would need some research to produce an exhaustive list.

Note that `google_application_credentials` above is something that *we invented* to allow a service account path to be configured permanently rather than relying on the potentially various different env var configurations or IAM roles that GCS clients support.

Reply:
I don't think so, this appears to be a valid environment variable for google. We also happen to intercept it first.
https://cloud.google.com/docs/authentication/application-default-credentials

Reply:
Added reference to the other options - attaching a service account and using gcloud CLI

![image](https://github.com/voxel51/fiftyone/assets/6363888/b5b2fcde-19d3-471d-9ee5-39f5abebbf81)